pipelines/short-read-alignment-vector/farm5/ShortReadAlignment.wdl

```diff
@@ -122,10 +147,8 @@ workflow ShortReadAlignment {
   }
 
   output {
-    Array[File] output_sams = ReadAlignment.output_sam
-    Array[File] output_bams = ReadAlignmentPostProcessing.output_bam
     File indel_realigner_interval_list_file = RealignerTargetCreator.output_interval_list_file
-    File output_bam = IndelRealigner.output_file
+    File output_bam = FixMateInformation.output_file
```

Comment:
Just want to confirm: is this the more correct bam to provide as output and does the spec need to be updated for this change?

Reply:
That was an error in the implementation currently on master.  The spec doesn't explicitly say which task generates the output bam, but FixMateInformation is the last task that changes the bam, and its output is the one that is used to generate the metrics generated by the pipeline so this should be correct.

Reply:
👍